src/pages/roadmap/javascript/index.css

```diff
@@ -0,0 +1,96 @@
+body {
```

Comment:
remove

```css
body {
    font-family: sans-serif;
    background-color: #f4f4f4;
  }
```

Reply:
you need to read carefully https://github.com/CodeHarborHub/codeharborhub.github.io/pull/2933#pullrequestreview-2172510159 I shared alreade for css

Reply:
for example you can check ou https://github.com/CodeHarborHub/codeharborhub.github.io/blob/main/src/components/Roadmap/Roadmap.css like this

Reply:
for example you can check ou https://github.com/CodeHarborHub/codeharborhub.github.io/blob/main/src/components/Roadmap/Roadmap.css like this